### DISCUSSION TITLE
Bug? Sometimes reactions disappear

### DISCUSSION THREAD

**Discussion by jeanlucaslima** [community/community#20521] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi, I love the new beta feed where I can react to releases from some repos, just one thing I am finding odd, I reacted to this release and whenever I come back to the feed my reactions disappeared: 

<img width="721" alt="image" src="https://user-images.githubusercontent.com/1311402/178984425-21dde510-8bb8-4a54-8459-90152474939d.png">

**Comment by Tostisto** [comment, marked as answer]
That's weired. You can report bug [here](https://support.github.com/contact/bug-report). 

For me is working normally

**Reply by jeanlucaslima** [reply]
Thanks! I wasn't aware of the bug report link.

**Reply by Tostisto** [reply]
Maybe the problem could be in your browser. Try clean cash or something like this.

**Reply by jeanlucaslima** [reply]
Yeah, very likely, my reaction appeared after some minutes.

**Comment by jfullstackdev** [comment]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
